docs/Contributing/API-for-contributors.md

```diff
@@ -541,6 +541,10 @@ The MDM endpoints exist to support the related command-line interface sub-comman
 - [Renew VPP token](#renew-vpp-token)
 - [Delete VPP token](#delete-vpp-token)
 - [Batch-apply MDM custom settings](#batch-apply-mdm-custom-settings)
+- [Batch-apply packages](#batch-apply-packages)
+- [Batch-apply App Store apps](#batch-apply-app-store-apps)
+- [Get token to download package](#get-token-to-download-package)
+- [Download package using a token](#download-package-using-a-token)
```

Comment:
@lucasmrod when we merged in reference docs for the 4.57 release, it looks like the batch-apply endpoints got moved to the REST API docs instead of the contributors docs.

I moved them back. When you get the chance, can you please check to see if these changes look right?

Also, I moved the download endpoints here b/c I think they're only for the Fleet UI to use (#21341). The best practice download endpoint for the IT admin to use in automations is here: https://fleetdm.com/docs/rest-api/rest-api#download-package

Reply:
I'm super confused. It seems we deliberately moved them from API for contributors to REST API? ([here](https://github.com/fleetdm/fleet/pull/21912)).

Reply:
Hmm, @rachaelshaw re your comment [here](https://github.com/fleetdm/fleet/pull/21912#issuecomment-2361363765), are we changing out philosophy? That is, endpoints that fleetctl uses live in API for contributors.

Reply:
@noahtalerman nope, I was just confused; I saw Luke's review comment about the other `/app_store_apps` endpoints being the REST API and was like "oh yeah this looks like it fits with those" — I should have double-checked how this endpoint was being used first!

Reply:
Got it!

> It seems we deliberately moved them from API for contributors to REST API?

@lucasmrod just following up here to say we goofed (which happens). We want the batch-apply endpoints to live in the API for contributors docs.

Reply:
@rachaelshaw I think we're ready for final review/merge.